app/jobs/upvs/drafts/load_content_job.rb

```diff
@@ -55,8 +55,7 @@ def load_message_draft_objects(message_draft, objects_path, signed:, to_be_signe
       )
     end
 
-    EventBus.publish(:message_thread_created, message_draft.thread) if message_draft.thread.previously_new_record?
-    EventBus.publish(:message_created, message_draft)
+    message_draft.publish_created_event
```

Comment:
neschovame tuto logiku schovat do wiringu kde proste ked posles message_thread_created tak to urobi aj iny event.

Reply:
Presunula som to, ale s touto castou nie som spokojna: https://github.com/slovensko-digital/govbox-pro/blob/e6182f975745bf72e9bdee78bfa679bc7b111a4e/app/lib/event_bus.rb#L24